README.md

```diff
@@ -1,7 +1,16 @@
-# Adobe Commerce Bold Checkout Integration.
-## Description
-This module replaces the native Adobe Commerce checkout with Bold Checkout.
-## Installation of the Adobe Commerce module
-https://developer.boldcommerce.com/default/guides/platform-integration/adobe-commerce/installation
-## Documentation
-How to install a module on your Adobe Commerce platform - https://experienceleague.adobe.com/docs/commerce-cloud-service/user-guide/configure-store/b2b-module.html?lang=en
+# Adobe Commerce Bold Checkout Integration
+
+This extension replaces the built-in Adobe Commerce checkout experience with Bold Checkout.
```

Comment:
Maybe to not scare our customer we should explain it "can replace" their native checkout, it can also live by the side in parallele of it or be simply disabled.

Reply:
This is mostly for people that will install the paypal branded flow where they will keep their native checkout, but they will have to install this module.

Reply:
Good point! I'll change that!